mockdjangosaml2/views.py

```diff
@@ -5,7 +5,7 @@
 from django.contrib import auth
 from django.contrib.auth.decorators import login_required
 from django.contrib.auth.forms import AuthenticationForm
-from django.contrib.auth.views import logout as django_logout
+from django.contrib.auth.views import LogoutView
```

Comment:
I think the logout was moved to `from django.contrib.auth import logout`

https://docs.djangoproject.com/en/2.2/topics/auth/default/#how-to-log-a-user-out